README.md

```diff
@@ -38,4 +67,196 @@ const plantUMLClassDiagram = appRelationsDiscovery.getRelations(...docs,{syntax:
 - For reactFlow nodes
 ```javascript
 const reactFlowNodes = appRelationsDiscovery.getRelations(...docs,{syntax:'reactFlow'});
-```
\ No newline at end of file
+```
+## Usage
+
+### Node.js
+
+```javascript
+const appRelationsDiscovery = require('@asyncapi/app-relations-discovery');
+const path = require('path');
+const fs = require('fs');
+
+const examplesPath = './test/examples/flightService';
+
+function getAsyncApiExamples() {
+  const docs = [];
+  const files = fs.readdirSync(examplesPath);
+  for (const file of files) {
+    const document_path = path.join('./examples/flightService', file);
+    const asyncApiDoc = fs.readFileSync(path.resolve(__dirname, document_path),'utf8');
+    docs.push(asyncApiDoc);
+  }
+  return docs;
+}
+
+// For default output syntax
+const defaultOutput = appRelationsDiscovery.getRelations(...docs);
+
+// For mermaid Flowchart 
+const mermaidFlowchart = appRelationsDiscovery.getRelations(...docs,{syntax:'mermaid'});
+
+// For plantUML classDiagram 
+const plantUMLClassDiagram = appRelationsDiscovery.getRelations(...docs,{syntax:'plantUML'});
+
+// For reactFlow nodes
+const reactFlowNodes = appRelationsDiscovery.getRelations(...docs,{syntax:'reactFlow'});
+```
+
+### Default Output Syntax
+
+// Explain:
+
+```javascript
+Map(n) {
+  '<server1>' => Map(m) {
+    'channel1' => { sub: [Map], pub: [Map] },
+    'channel2' => { sub: [Map], pub: [Map] }
+  }
+}
+```
+## Mermaid Flowchart
+### Syntax
+Based on FlightService example.
+```
+graph TD
+ server1[(mqtt://localhost:1883)]
+FlightMonitorService[Flight Monitor Service]
+FlightMonitorService -- flight/update --> server1
+FlightNotifierService[Flight Notifier Service]
+server1 -- flight/update --> FlightNotifierService
+FlightSubscriberService[Flight Subscriber Service]
+FlightSubscriberService -- flight/queue --> server1
+server1 -- flight/queue --> FlightMonitorService
+```
+
+### Flowchart
+
+<img src="./public/img/mermaid.png" alt="Mermaid Flowchart" width="500"/> </br></br>
+
+### Link
+
+You can paste the mermaid output syntax to visualize the relations as Flowchart here: https://mermaid-js.github.io/mermaid-live-editor/edit
```

Comment:
I think editor supports you to put all data encoded in the url so you can already provide a direct link here. Also, don't call it `Link` section but just mention below code snippet with syntac